helper/helics-helper.cc

```diff
@@ -80,6 +80,13 @@ HelicsHelper::SetupApplicationFederate(void)
   GlobalValue::Bind ("SimulatorImplementationType", StringValue ("ns3::HelicsSimulatorImpl"));
 }
 
+void
+HelicsHelper::SetupApplicationFederateWithConfig(std::string &configFileName)
+{
+	helics_federate = std::make_shared<helics::MessageFederate> (configFileName);
```

Comment:
```suggestion
	helics_federate = std::make_shared<helics::CombinationFederate> (configFileName);
```

I think this should just be a CombinationFederate so that the ValueFederate interface is also accessible to users that want to use it.

Reply:
@nightlark I understand your suggestion, but, since we are in the ns-3 realm, would there ever be a ValueFederate? My understanding was that value federates were for physical connection, while, if we go through ns-3, it means federates will exchange messages. However, if you want me to give it a test, I can do that.

Reply:
There could be -- a lot of projects treat ns-3 as just an intermediary for exchanging messages coming directly from HELICS simulators, but there are a few that are modeling "devices" in ns-3 as if they had a physical connections to read values and interact with things outside ns-3.

One project I know of is modeling sensors as ns-3 devices/applications, and they are using the value interface for the ns-3 "sensor" application to publish values and subscribe to values in an external transmission/distribution simulator.

Reply:
Sounds good. I will make the change on my end and test it. I am using JSON-type messages between GridLAB-D and Python federate traveling through ns-3 network, and after running into many issues setting it up and running it, I want to make sure such a small innocent change won't cause a storm. :-)